src/usr/bin/google_set_multiqueue

```diff
@@ -256,6 +271,13 @@ fi
 # IRQ binding for numa 0, CPUs [0, 51] and [104, 155] are for numa 0.
```

Comment:
Can you add a comment here-ish (somewhere in the A3 platform section) explaining that we are relying on mlx5 drivers to set IRQ for CX-7 and other non-gvnics?

Otherwise lgtm